python/tk_houdini_renderman/farm_dialog.py

```diff
@@ -55,24 +57,60 @@ def __init__(
         self.submission_name.setMinimumSize(300, 0)
         layout.addWidget(self.submission_label)
         layout.addWidget(self.submission_name)
+        layout.addSpacing(8)
 
         self.priority_label = QtWidgets.QLabel("Priority")
-        self.priority = QtWidgets.QLineEdit(priority)
+        self.priority = QtWidgets.QDoubleSpinBox()
+        self.priority.setDecimals(0)
+        self.priority.setRange(0, 100)
+        self.priority.setValue(priority)
         layout.addWidget(self.priority_label)
         layout.addWidget(self.priority)
+        layout.addSpacing(8)
 
+        self.frames_group = QtWidgets.QWidget()
+        frames_group_layout = QtWidgets.QHBoxLayout()
+        frames_group_layout.setContentsMargins(0, 0, 0, 0)
+
+        self.frame_range = QtWidgets.QWidget()
+        frame_range_group_layout = QtWidgets.QVBoxLayout()
+        frame_range_group_layout.setContentsMargins(0, 0, 4, 0)
         self.framerange_label = QtWidgets.QLabel("Frame Range")
         self.framerange = QtWidgets.QLineEdit(framerange)
-        layout.addWidget(self.framerange_label)
-        layout.addWidget(self.framerange)
+        frame_range_group_layout.addWidget(self.framerange_label)
+        frame_range_group_layout.addWidget(self.framerange)
+        self.frame_range.setLayout(frame_range_group_layout)
+        frames_group_layout.addWidget(self.frame_range)
+
+        self.fpt = QtWidgets.QWidget()
```

Comment:
Pep8 formatting issue, might wanna rename `fpt` to a more readable variable name like `frames_per_task`